wolox-books/src/app/screens/auth/screens/book-list/book-list.component.html

```diff
@@ -1,3 +1,7 @@
-<p>
-  book-list works!
-</p>
+<div class="books">
+  <div class="book" *ngFor="let book of books">
+    <img [src]="book.image_url" class="book-img" alt="Book cover">
```

Comment:
The `alt` is an attribute that must explain the image, not be something generic Maybe you can use the book title